certified-connectors/Tallyfy/apiProperties.json

```diff
@@ -5,7 +5,7 @@
         "type": "oauthSetting",
         "oAuthSettings": {
           "identityProvider": "oauth2",
-          "clientId": "ty-oauth2-client-id",
+          "clientId": "15",
```

Comment:
@adheegm If you have changed 'client-id' value, please let us know, there is a special secure process to accept changes in client credentials.  And in the open-source repo, it is expected to be a dummy value for security reasons, like below.

"clientId": "[[DUMMY]]",

Reply:
@sriyen-msft Thanks for letting me know - I've changed it. Thanks